indra/newview/skins/default/xui/en/notifications.xml

```diff
@@ -4252,6 +4252,20 @@ Unable to add estate manager [AGENT] to banned list.
 Can&apos;t change appearance until clothing and shape are loaded.
   </notification>
 
+  <notification
+   icon="alertmodal.tga"
+   name="UsavedWearableChanges"
+   type="alertmodal">
+You have unsaved changes.
+      <tag>group</tag>
+    <tag>confirm</tag>
+    <usetemplate
+     ignoretext="Confirm before I discard unsaved wearable changes"
```

Comment:
```suggestion
     ignoretext="Are you sure you want to discard unsaved wearable changes?"
```

This will probably fit better.

Reply:
That's not a visible part, it's for preferences so it should cater to preferences. Popup shows a basic 'don't ask again' text.
![image](https://github.com/user-attachments/assets/856d250a-b0cd-4d97-b603-adeee7ccd88e)

Reply:
Ah, that makes sense, thank you.